code/src/snam_arm.cpp

```diff
@@ -0,0 +1,215 @@
+#include "snam_arm.hpp"
```

Comment:
#include "snam_arm.hpp" -> #include <snam_arm.hpp>

Voor alle includes wordt <> gebruikt dit staat niet in de style guide maar werd ik persoonlijk tijdens het project op gewezen.

Reply:
I changed all the includes to use <>: https://github.com/R2D2-2019/robot_arm/pull/28/commits/5c7c5881b498883b6b19151a51e7383eedddde70